installer/lib/mix/tasks/phx.new.ex

```diff
@@ -78,6 +78,11 @@ defmodule Mix.Tasks.Phx.New do
 
   You can read more about umbrella projects using the
   official [Elixir guide](http://elixir-lang.org/getting-started/mix-otp/dependencies-and-umbrella-apps.html#umbrella-projects)
+  
+  Prints Phoenix version (`-v`, `--version`).
```

Comment:
It prints the installer version, not to be confused with version of phoenix you may have in your project.

Reply:
Thank you for clarification. I will create new PR.